.circleci/config.yml

```diff
@@ -114,16 +114,17 @@ jobs:
       - node/install-packages:
           pkg-manager: yarn
           cache-key: "yarn.lock"
+
   linters:
     docker:
       - image: 'cimg/ruby:2.7-node'
     resource_class: small
     steps:
       - checkout
       - ruby/install-deps
-      - ruby/rubocop-check:
-          format: progress
-          label: Inspecting with Rubocop
+#      - ruby/rubocop-check:
+#          format: progress
+#          label: Inspecting with Rubocop
```

Comment:
Leaving Rubocop linting out?

Reply:
The agreement on the convo that followed in slack, is that we're better leaving these out for now, until we can address the linting violations, so that the test is more useful than being red every time until then.